gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineFormat.java

```diff
@@ -49,6 +52,10 @@ public void apply(Project project) {
         if (project == project.getRootProject()) {
             if (BaselineFormat.palantirJavaFormatterState(project) == FormatterState.ON) {
                 project.getPluginManager().apply(PalantirJavaFormatIdeaPlugin.class);
+                // Also, configure a specific version of this dependency.
+                // To pick up updates, users should bump this using a constraint in their versions.props.
+                project.getDependencies()
```

Comment:
I thought that this was handled by the PJF plugin? does it make sense to move it over so that baseline just does super basic plumbing?

Reply:
A default is currently provided by the PJF plugin, the idea here was to override that and provide a pinned version.
Discussed with Dan and we're gonna go with a different approach (will post a comment here).